### PR TITLE
[5.0] Set mail config values in .env file.

### PR DESCRIPTION
**Pull request by martinbean**
Make the mail configuration values look in **.env** file by default.

### CODE REVIEW

File: .env.example [hunk 1]
```diff
@@ -8,4 +8,5 @@ DB_USERNAME=homestead
 DB_PASSWORD=secret
 
 CACHE_DRIVER=file
+MAIL_DRIVER=file
```
Comment (crynobone) [inline .env.example]: file mail driver?

Comment (GrahamCampbell) [inline .env.example]: lol

Comment (martinbean) [inline .env.example]: @crynobone @GrahamCampbell Yeah, that’s a typo :blush:

Fixed! Thanks for pointing that out.

**Comment by TheShiftExchange** [comment]
I like this PR. But I think the naming convention is wrong.

Instead of names like `SMTP_USERNAME` - it should be `MAIL_USERNAME` - because you might not be using an SMTP driver.

**Comment by martinbean** [comment]
@theshiftexchange Thanks for the feedback. I’m named them as such as the comment blocks above each variable explicitly mention they’re the SMTP host/port/username/password configuration values.

Happy to rename them if these credentials are used in a non-SMTP instances, but can’t think of any of those instances off the top of my head.

**Comment by TheShiftExchange** [comment]
No problems - its just a thought.

What about since you do not have a default username+password in the config `env()` - these should also probably be included in the default `.env` file since you are expected to set them?

**Pull request merged**